app/uk/gov/hmrc/gform/views/hardcoded/CompositeAuthFormPage.scala

```diff
@@ -51,7 +53,10 @@ class CompositeAuthFormPage(
     )
   }
 
-  val hasErrors: Boolean = errorSummary.errorList.nonEmpty
+  val hasErrors: Boolean =
+    if (se.asString === SuppressErrors.seNo)
```

Comment:
Descending to `String` level is quite ugly, what about implementing `Eq[SupressError]` typeclass instance instead?